skrub/tests/test_interpolation_join.py

```diff
@@ -76,117 +91,63 @@ def transform(self, X):
     assert_array_equal(ns.col(join, "B"), [0, 1])
 
 
-def test_no_multioutput(df_module, buildings, weather):
-    if df_module.name == "polars":
-        pytest.xfail(
-            reason=(
-                "In polars, DataFrame.drop() got an unexpected keyword argument 'axis'."
-            )
-        )
-    weather = df_module.DataFrame(weather)
-    buildings = df_module.DataFrame(buildings)
-    transformed = InterpolationJoiner(
-        weather,
-        main_key=("latitude", "longitude"),
-        aux_key=("latitude", "longitude"),
-    ).fit_transform(buildings)
-    assert ns.shape(transformed) == (2, 5)
-
-
-def test_condition_choice(df_module):
-    if df_module.name == "polars":
-        pytest.xfail(
-            reason=(
-                "In polars, DataFrame.drop() got an unexpected keyword argument 'axis'."
-            )
-        )
+def test_duplicate_column(df_module):
     main = df_module.make_dataframe({"A": [0, 1, 2]})
-    aux = df_module.make_dataframe({"A": [0, 1, 2], "rB": [2, 0, 1], "C": [10, 11, 12]})
+    aux = df_module.make_dataframe({"A": [0, 1, 2], "C": [10, 11, 12]})
     join = InterpolationJoiner(
         aux, key="A", regressor=KNeighborsRegressor(1)
     ).fit_transform(main)
     assert_array_equal(ns.to_list(ns.col(join, "C")), [10, 11, 12])
 
-    join = InterpolationJoiner(
```

Comment:
here why is this check not necessary anymore?

Reply:
We were just fitting on two different column names ("A" and "rB"). Now the test checks that the double fitting returns unique column names. Other tests check fitting on left + right keys